javascript/ql/src/javascript.qll

```diff
@@ -75,6 +75,7 @@ import semmle.javascript.frameworks.DigitalOcean
 import semmle.javascript.frameworks.Electron
 import semmle.javascript.frameworks.Files
 import semmle.javascript.frameworks.Firebase
+import semmle.javascript.frameworks.typeahead
```

Comment:
I think we want this spelled as `Typeahead`, `jQuery` is a special case. Also, this list of imports happens to be sorted.